jekyll/_cci2/env-vars-built-in.adoc

```diff
@@ -0,0 +1,19 @@
+---
+version:
+- Cloud
+- Server 3.x
+- Server 4.x
+---
+= Built-in Environment Variables
```

Comment:
For new pages could you make the filename the same as the title of the page so the URLs match the titles? I know this is going to mess up all the link changes you've made 😬 but I guess it's just a find and replace thing?